modules/sdk-core/src/bitgo/wallet/wallet.ts

```diff
@@ -1357,14 +1357,13 @@ export class Wallet implements IWallet {
   }
 
   /**
-   * Gets the user key chain for this wallet
+   * Gets the user keychain for this wallet
    *
-   * The user key chain is the first keychain of the wallet and usually has the encrypted prv stored on BitGo.
+   * The user keychain is the first keychain of the wallet and usually has the encrypted prv stored on BitGo.
    * Useful when trying to get the users' keychain from the server before decrypting to sign a transaction.
-   * @param params
    */
-  async getEncryptedUserKeychain(params: Record<string, never> = {}): Promise<{ encryptedPrv: string }> {
```

Comment:
Does this change mean that the following are to be expected in the returned object?
```  id: string;
  pub?: string;
  prv?: string;
  provider?: string;
  encryptedPrv?: string;
  derivationPath?: string;
  derivedFromParentWithSeed?: string;
  commonPub?: string;
  commonKeychain?: string;
  keyShares?: ApiKeyShare[];
  walletHSMGPGPublicKeySigs?: string;
  type: KeyType;
  source?: SourceType;
```
from Keychain?

Reply:
Yup! It's returning the keychain from the wp api.

Reply:
Actually not 100% on some of these fields... I don't think the following are valid fields that returned from the api: 
- keyShares
-  walletHSMGPGPublicKeySigs
will double check.

Reply:
Just double checked, those fields are only returned during the key creation workflows. 

Its a little unfortunate, but the `GET` and `POST` codecs in wp re-use the same response even though this is not valid. The `GET` call will return less data as the `POST` since the `POST` call has some ephemeral data. 

According to the codec, this type for this SDK call is valid, but ik its wrong. Ill create a ticket to address the codec issue, and then a separate ticket to fix the SDK. 

- https://bitgoinc.atlassian.net/browse/WP-1283
- https://bitgoinc.atlassian.net/browse/WP-1284

Reply:
Awesome, thanks!